libs/model/src/aggregates/contest/DeleteContestManagerMetadata.command.ts

```diff
@@ -0,0 +1,38 @@
+import { InvalidState, type Command } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { models } from '../../database';
+import { authRoles } from '../../middleware';
+import { mustExist } from '../../middleware/guards';
+
+const Errors = {
+  ContestNotCancelledOrEnded:
+    'Contest Manager must be cancelled or ended to delete',
+};
+
+export function DeleteContestManagerMetadata(): Command<
+  typeof schemas.DeleteContestManagerMetadata
+> {
+  return {
+    ...schemas.DeleteContestManagerMetadata,
+    auth: [authRoles('admin')],
+    body: async ({ payload }) => {
+      const contestManager = await models.ContestManager.findOne({
+        where: {
+          community_id: payload.community_id,
+          contest_address: payload.contest_address,
+        },
+      });
+      mustExist('Contest Manager', contestManager);
+
+      if (!contestManager.cancelled && !contestManager.ended) {
+        throw new InvalidState(Errors.ContestNotCancelledOrEnded);
+      }
+
+      contestManager.deleted_at = new Date();
```

Comment:
Do we want to automatically ignore deleted contest managers from Sequelize native queries? If so should set `paranoid: true` in the model. If you do set paranoid then can update this to `await contestManager.destroy()`